filebeat/util/data.go

```diff
@@ -0,0 +1,65 @@
+package util
+
+import (
+	"github.com/elastic/beats/filebeat/input/file"
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type Data struct {
+	Event common.MapStr
+	state file.State
+	Meta  Meta
+}
+
+type Meta struct {
+	Pipeline string
+	Fileset  string
+	Module   string
+}
+
+func NewData() *Data {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewData should have comment or be unexported